src/diffcalc_API/config.py

```diff
@@ -1,7 +1,27 @@
+import logging
+
 from pydantic import BaseSettings
 
 from ._version_git import __version__
 
+logging.basicConfig(
+    level=logging.DEBUG, format="[%(asctime)s] %(levelname)s:%(message)s"
```

Comment:
It would be good to pass this in as an argument to the program, or to use an environment variable. You don't really want the production instance at debug level as it will create lots of log spam, for example.

Reply:
Yep, good idea. I could add this in the helm chart maybe, and then anyone installing it can modify it as they wish